docs/udf.rst

```diff
@@ -279,8 +279,46 @@ instead of the original digital number range (thousands):
 
 Illustration of data chunking in ``apply`` with a  UDF
 ========================================================
+Chunk_polygon can be used to apply a specific kind of process to spatial chunks of a data cube. In the following example, we repeat the method of rescaling a data cube using chunk_polygon. Furthermore, in this example, we show that with the Geojson file, users can try with shapefile if they want to and fed in as a feature collection. Though a function to read JSON is shown here, you can replace it with your process.
```

Comment:
avoid putting a whole paragraph on a single line, this makes it hard to review/maintain

e.g see https://github.com/sembr/specification

Reply:
Updated